### PR TITLE
T344890: Add the use case application

### PR DESCRIPTION
**Pull request by protsack-stephan**
- add default set of components
- add libraries for all API calls
- add unit tests for API libraries and components
- add deployment instructions to README file

### CODE REVIEW

File: src/components/KnowledgePanel.vue [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  }
+}
+
+.wme-app-knowledge-panel-fact-name {
+  margin-right: 5px;
+}
+
+.wme-app-knowledge-panel-image {
+  width: 100%;
+  min-height: 250px;
```
Comment (ruairi-wmf) [inline src/components/KnowledgePanel.vue:101]: A minor suggestion, the image looks badly cropped for some portraits.  I think a lot of portrait images are vertical and the CSS is for horizontal crop.  

Maybe see if one of these can deal with vertical and horizontal images: `object-fit: contain;` or `object-fit: cover;` to resize the image and keep the aspect ratio or to zoom in and cover the div.

Comment (protsack-stephan) [reply src/components/KnowledgePanel.vue:101]: We do use  `object-fit: cover;` on the underlying image, this is just a wrapper `div` for that `img` tag. The problem here that the images are displayed not cropped and in their native resolution, so we are trying to feet all of them in a single format, that's why they are sometimes not looking good.

**Comment by prabhat393** [comment]
Looks good.

**Review by resquito — APPROVED** [review]
looks good

**Pull request merged**